mkdocs.yml

```diff
@@ -125,123 +126,129 @@ plugins:
-    - Memory: sessions/memory.md
-    - Vertex AI Express Mode: sessions/express-mode.md
-  - Callbacks:
-    - callbacks/index.md
-    - Types of callbacks: callbacks/types-of-callbacks.md
-    - Callback patterns: callbacks/design-patterns-and-best-practices.md
-  - Artifacts:
-    - artifacts/index.md
-  - Events:
-    - events/index.md
-  - Context:
-    - context/index.md
-    - Context compression: context/compaction.md
-  - Observability:
-    - Logging: observability/logging.md
-    - Cloud Trace: observability/cloud-trace.md
-    - AgentOps: observability/agentops.md
-    - Arize AX: observability/arize-ax.md
-    - Phoenix: observability/phoenix.md
-    - W&B Weave: observability/weave.md
-  - Evaluate:
-    - evaluate/index.md
-    - Criteria: evaluate/criteria.md
-  - MCP:
-    - mcp/index.md
-  - Plugins:
-    - plugins/index.md
-  - Bidi-streaming (live):
-    - streaming/index.md
-    - Custom Audio Bidi-streaming app sample (WebSockets): streaming/custom-streaming-ws.md
-    - Bidi-streaming development guide series: streaming/dev-guide/part1.md
-    - Streaming Tools: streaming/streaming-tools.md
-    - Configurating Bidi-streaming behaviour: streaming/configuration.md
-    - Google ADK + Vertex AI Live API (blog post): https://medium.com/google-cloud/google-adk-vertex-ai-live-api-125238982d5e
-  - Grounding:
-    - Understanding Google Search Grounding: grounding/google_search_grounding.md
-    - Understanding Vertex AI Search Grounding: grounding/vertex_ai_search_grounding.md
-  - Safety and Security: safety/index.md
-  - A2A Protocol:
-    - a2a/index.md
-    - Introduction to A2A: a2a/intro.md
-    - A2A Quickstart (Exposing): a2a/quickstart-exposing.md
-    - A2A Quickstart (Consuming): a2a/quickstart-consuming.md
-    - A2A Protocol Documentation: https://a2a-protocol.org
-  - Community Resources: community.md
-  - Contributing Guide: contributing-guide.md
-  - API Reference:
-    - api-reference/index.md
-    - Python ADK: api-reference/python/index.html
-    - Java ADK: api-reference/java/index.html
-    - CLI Reference: api-reference/cli/index.html
-    - Agent Config reference: api-reference/agentconfig/index.html
-    - REST API: api-reference/rest/index.md
+    - Agent Runtime:
+      - Agent Runtime: runtime/index.md
+      - Runtime Config: runtime/runconfig.md
+      - Resume Agents: runtime/resume.md
+    - Deploy:
+      - deploy/index.md
+      - Agent Engine: deploy/agent-engine.md
+      - Cloud Run: deploy/cloud-run.md
+      - GKE: deploy/gke.md
+    - Observability:
+      - Logging: observability/logging.md
+      - Cloud Trace: observability/cloud-trace.md
+      - AgentOps: observability/agentops.md
+      - Arize AX: observability/arize-ax.md
+      - Phoenix: observability/phoenix.md
+      - W&B Weave: observability/weave.md
+    - Evaluate:
+      - evaluate/index.md
+      - Criteria: evaluate/criteria.md
+    - Safety and Security:
+      - safety/index.md
+  - Components:
```

Comment:
based on the relative importance of the components, I think the order should be:

- Technical overview
- Context
- Sessions & Memory
- Events
- Artifacts
- Callbacks
- Plugins
- MCP
- Agent2Agent (A2A)
- Streaming (Live Bidi)
- Grounding